templates/home/home.html

```diff
@@ -1,365 +1,220 @@
+            <h3 class="mt-4 text-2xl font-bold sm-text-xl">WHEN</h3>
+            <p class="mt-2 sm-leading-relaxed">{{ CONFERENCE_DATE }}</p>
           </div>
         </div>
-        <!-- End Icon Blocks -->
-   
       </div>
-      </section>
-   <!-- Black Call To Action --> 
-
-
+    </div>
+  </div>
+</section>
 
-    <!-- Our programme News -->
-    <section class="container g-pt-100 g-pb-60">
-      <!-- Heading -->
-      <div class="row justify-content-center text-center g-mb-50">
-        <div class="col-lg-9">
-          <h2 class="h2 g-color-black g-font-weight-600 mb-2">Our Programme</h2>
-          <div class="d-inline-block g-width-30 g-height-2 g-bg-primary mb-2"></div>
-          <p class="lead mb-0"> </p>
+<section class="bg-light py-5 tickets">
+  <div class="container mx-auto px-4 py-16 lg-h-60">
+    <div class="row justify-content-center align-items-center">
+      <div class="col-12 col-lg-6 text-center">
+        <h2 class="fs-1 fw-bold text-dark text-uppercase">
+          Buy Tickets
+        </h2>
+        <p class="fs-5 fw-medium mb-0">
+          Tickets can be bought by following the link below.
+        </p>
+        <div class="py-5">
+          <a
+            href="{{ QUICKET_LINK }}"
+            class="btn text-uppercase text-white px-5 py-3 fw-bold w-50 w-md-auto"
+            target="_blank"
+            style="border-radius: 0.375rem; min-width: 16rem;">
+            Buy Tickets
+          </a>
         </div>
       </div>
-      <!-- End Heading -->
- 
-      <div class="card-group d-block d-md-flex g-mx-minus-15">
-        <div class="card u-shadow-v36 g-brd-none g-mx-15 col-md-4  g-mb-30">  
-          <!-- Research Article -->
-          <article>
-            <div class="card-body g-bg-white text-center g-pa-20">
-              <h3 class="h4 mb-0">Tutorials/Workshops</h3>
-            </div>
-             
-          </article>
-          <!-- End Research Article -->
-        </div>
-
+    </div>
+  </div>
+</section>
 
-        
-        <div class="card u-shadow-v36 g-brd-none g-mx-15 col-md-4  g-mb-30 ">
-          <!-- Research Article -->
-          <article>
-            <div class="card-body g-bg-white text-center g-pa-20">
-              <h3 class="h4 mb-0">Main Conference days.</h3>
-            </div>
-            
-          </article>
-          <!-- End Research Article -->
+<section class="sponsor-section">
+  <div class="sponsor-container">
+    <!-- Sticky Sponsor Call-to-Action -->
+    <aside class="sponsor-cta">
+      <h1 class="sponsor-title">Sponsors!</h1>
+      <p class="sponsor-description">
+        With over 200+ attendees expected, this is your chance to boost brand visibility, connect with top Python professionals, generate leads, and showcase your commitment to the Python community. Encourage your company to sponsor and be a part of this dynamic event!
+      </p>
+      <a href="/2024/sponsors/why-sponsor" target="_blank" class="sponsor-button">Sponsor Us!</a>
```

Comment:
amazing stuff @Leon12-aaryson :clap: well done

could we please include propell here